### DISCUSSION TITLE
Trang web không vào được

### DISCUSSION THREAD

**Discussion by xungvu2k1** [d2l-ai/d2l-vi#3132] [Q&A] [answered]
Em đã thử vào trang web nhưng không vào được ạ.

**Comment by trgiangdo** [comment]
Bạn có thể truy cập theo đường dẫn mới này nhé.
https://vi.d2l.ai/

**Comment by xungvu2k1** [comment]
em cảm ơn anh ạS2


Vào Th 5, 22 thg 9, 2022 vào lúc 22:46 Đỗ Trường Giang <
***@***.***> đã viết:

**Comment by thanhcsf** [comment, marked as answer]
Chào bạn @xungvu2k1,

Nhờ bạn xem qua bản PDF có thể tải về tại đây nhé: https://github.com/d2l-ai/d2l-vi/releases/tag/v0.14.4

Về phiên bản web, chúng tôi sẽ kiểm tra lại và phản hồi đến bạn sau nhé.

Ps: phiên bản ở trên chưa được cập nhật nên chúng tôi chưa công khai, vì vậy để đảm bảo về chất lượng nội dung, bạn xem bản PDF giúp nhóm nhé.

Cảm ơn bạn.

**Reply by thanhcsf** [reply]
Chào bạn @xungvu2k1,

Bạn có thể vào website này để xem trực tuyến rồi nha: https://d2l.aivivn.com/

Cảm ơn bạn.

**Comment by xungvu2k1** [comment]
Em cảm ơn tất cả mọi người nhiều nhé.

Vào 13:13, T.2, 26 Th9, 2022 Duy–Thanh Doan ***@***.***> đã
viết:
